pages/index.js

```diff
@@ -1,12 +1,42 @@
-import { useEffect } from 'react';
-import { useRouter } from 'next/router';
-import Layout from '@/components/Layout';
+import { useEffect } from "react";
+import { useRouter } from "next/router";
+import Layout from "@/components/Layout";
+import BannerLogo from "../public/Hack-R-Play/BannerLogo.png";
+import Banner from "@/components/Banner";
+import PastInitialives from "@/components/PastInitialives";
+
 
 export default function Home() {
 	const router = useRouter();
-	useEffect(() => {
-		router.push('/hackrplay/2022/home');
-	}, []);
+	// useEffect(() => {
+	// 	router.push('/hackrplay/2022/home');
+	// }, []);
+	const recentEvent={
+		name: '#2PlaysAMonth',
+		description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Corporis, debitis?'
+	}
```

Comment:
We will get this array of event objects from DB in the future. For now, you can hardcode